doc/particles.md

```diff
@@ -362,7 +373,9 @@ is achieved by encapsulating small pieces of behavior in every of particles and
 Two entities allowing `Particle` to nest each other are: `SingleChildParticle` mixin and `ComposedParticle` class.
```

Comment:
The description above needs updating, but I'm not allowed to comment that far from a change

Reply:
Uh how u mean?